Warnings in RDebugSnap

@RKX1209 this is yours:
```
snap.c: In function ‘print_hash’:

snap.c:167:3: warning: format ‘%llx’ expects argument of type ‘long long unsigned int’, but argument 3 has type ‘int’ [-Wformat]

snap.c: In function ‘r_debug_snap’:

snap.c:244:2: warning: return makes integer from pointer without a cast [enabled by default]
```

--- a/libr/debug/snap.c
+++ b/libr/debug/snap.c
@@ -164,7 +164,7 @@ R_API int r_debug_snap_set_idx(RDebug *dbg, int idx) {
 static void print_hash(ut8 *hash, int digest_size) {
 	int i = 0;
 	for (i = 0; i < digest_size; i++) {
-		eprintf ("%02"PFMT64x, hash[i]);
+		eprintf ("%02"PFMT32x, hash[i]);
 	}
 	eprintf ("\n");
 }

